PRODUCTS.md

```diff
@@ -36,34 +36,44 @@ or any damage to yourself or your products.
 IKEA has following products and product categories for the Smart Home series
 (also called TRADFRI):
 
-* Smart Home Gateway
-* FLOALT (light panel)
-* GUNNARP (ceiling/wall lamp)
-* LEPTITER (spotlight)
-* IRSTA  (worktop lighting)
-* OMLOPP (spotlight)
-* STRÖMLINJE (worktop lighting)
-* SKYDRAG (worktop lighting)
-* FYRTUR / KADRYL (window blinds)
-* 5-button remote control (V1 and V2)
-* 2-button dimmer/switch
-* Wheel dimmer
-* Shortcut button
-* Motion detector (V1 and V2)
-* Signal repeater + battery charger
-* Switching outlet plug/mains
-* LED driver 10 W with 3 connections
-* LED Driver 30 W
-* Bulb E27 1000 lm white/warmwhite
-* Bulb E27 800 lm warmwhite
-* Bulb E27 800 lm white/warmwhite transparent
-* Bulb E27 600 lm color
-* Bulb E14 600 lm color
-* Bulb E14 600 lm white/warmwhite
-* Bulb E27 250 lm filament gold
-* Bulb GU10 400 lm warmwhite
-* Bulb GU10 400 lm white/warmwhite
-* Bulb GU10 350 lm color
+ Name   | Product ID | data | comment
```

Comment:
I rather have the Product ID first, then the name. Comment and data is unnecessary IMHO, because that is for the bottom parts.

Can you also format the table according to [this](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/organizing-information-with-tables).